doc/develop/backend-api.rst

```diff
@@ -237,20 +242,46 @@ a single component. That is why Scanner inherit from Emitter.
 
-    .. py:attribute:: translation
+    .. py:attribute:: scale
 
-        *(VA, tuple of floats, unit=px)* How much shift is applied to the center of the area acquired. It is expressed in pixels (the size of a pixel being defined by pixelSize, and so independent of .scale).
+        *(VA, tuple of floats or int, unit=ratio)* ratio of the size of the scannable area divided by the size of the scanned area.
+        By convention, (1, 1) corresponds to the closest distance between pixels centers while scanning at the highest resolution.
+        Note that this is the inverse of the typical definition of scale (i.e., increasing the scale leads to a smaller scanned area).
+        The advantage of this definition is that its meaning is very similar to binning.
+        Note that the MD_PIXEL_SIZE metadata of a dataflow will depend both on pixelSize and scale (i.e., MD_PIXEL_SIZE = pixelSize * scale).
 
-    .. py:attribute:: scale
+    .. py:attribute:: translation
 
-        *(VA, tuple of floats or int, unit=ratio)* ratio of the size of the scannable area divided by the size of the scanned area. Note that this is the inverse of the typical definition of scale (i.e., increasing the scale leads to a smaller scanned area). The advantage of this definition is that its meaning is very similar to binning. Note that the MD_PIXEL_SIZE metadata of a dataflow will depend both on pixelSize and scale (i.e., MD_PIXEL_SIZE = pixelSize * scale).
+        *(VA, tuple of floats, unit=px)* How much shift is applied to the center of the area acquired.
+        It is expressed in pixels (the size of a pixel being defined by pixelSize, and so independent of .scale).
+        It is limited by the field of view of the scanner, and it is automatically
+        clipped based on the current resolution and scale.
+        A bigger value in X means a shift towards the right, and a bigger value in Y means a shift towards the bottom.
+        (0, 0) means that it's at the center of the scan area.
+        The MD_POS in the image metadata contains an adjusted MD_POS to indicate the center of the image,
+        based on the MD_POS in the scanner metadata and the pixel size.
 
     .. py:attribute:: rotation
 
         *(VA, float, unit=rad)* counter-clockwise rotation to apply on the original area to obtain the actual area to scan.
 
     .. Rationale: we could have done slightly differently by using a general .transformation (VA, array of float, shape of (3,3) for a 2D resolution). It would have been a transformation matrix from the scanning area to the actual value. Very generic, but more complex to use and read and the advanced transformations possible don't seem to be useful.
 
+    .. py:attribute:: shift
+
+        *(VA, tuple of floats, unit=m, optional)* How much shift is applied to the center of the area acquired.
+        It is expressed in meters.
+        A bigger value in X means a shift towards the left (ie, the center of the new image corresponds to a point
```

Comment:
I am not sure what is meant be this line in the bracket (e, the center of the new image corresponds to a point)

Reply:
point -> position. "the center of the new image corresponds to a position at the left of the center of the previous image". Is that clear enough? Or how would you say it?

Reply:
It is clear for me. I did not read the next line as I thought it ended at point.